facebook/OpenBIC
# fby4: ff: Switch PMIC and DIMM MUX after checking CXL is ready

# Description:
	After ten seconds of power on, if CXL heartbeat is at the normal frequency, BIC will switch the PMIC and DIMM MUX from CXL to BIC.

# Motivation:
	According to the circuit design, VR and DIMM should be switched to BIC after CXL is ready.
# Test plan:
	1. Check DIMM can be scanned by I2C: Pass
	2. Check VR can be scanned by I2C: Pass
# Test log:
1. Check VR address(0x72, 0x76):
	uart:~$ i2c scan I2C_8
		 0  1  2  3  4  5  6  7  8  9  a  b  c  d  e  f
	00:             -- -- -- -- -- -- -- -- -- -- -- --
	10: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --
	20: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --
	30: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --
	40: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --
	50: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --
	60: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --
	70: -- -- 72 -- -- -- 76 --
	2 devices found on I2C_8

2. Check DIMM address(0x50~0x53):
	uart:~$ i2c scan I2C_9
		 0  1  2  3  4  5  6  7  8  9  a  b  c  d  e  f
	00:             -- -- -- -- -- -- -- -- -- -- -- --
	10: -- -- -- -- -- -- -- -- 18 19 1a 1b -- -- -- --
	20: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --
	30: -- -- -- -- -- -- 36 37 -- -- -- -- -- -- -- --
	40: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --
	50: 50 51 52 53 -- -- -- -- -- -- -- -- -- -- -- --
	60: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --
	70: -- -- -- -- -- -- -- --
	10 devices found on I2C_9
